dist/rpm/openQA.spec

```diff
@@ -419,6 +434,8 @@ done
 #
 install -D -m 644 /dev/null %{buildroot}%{_localstatedir}/log/openqa
 install -m 0644 %{_sourcedir}/openQA.changes %{buildroot}%{_datadir}/openqa/public/Changelog
+install -Dm 0644 completion/openqa-cli-completion.bash %{buildroot}%{_datadir}/bash-completion/completions/openqa-cli
+install -Dm 0644 completion/openqa-cli-completion.zsh %{buildroot}%{_datadir}/usr/share/zsh/site-functions/_openqa-cli
```

Comment:
sorry

Reply:
Apart from missing a slash, now I think _sourcedir might not be the directory we want here.
Try it out by adding the slash.
Otherwise you can just move those statements up before the `cd` statement.